multiworld/core/image_env.py

```diff
@@ -42,9 +42,8 @@ def __init__(
             # init_camera(viewer.cam)
             # sim.add_render_context(viewer)
         self._render_local = False
-        self._img_goal = None
-
         img_space = Box(0, 1, (self.image_length,))
+        self._img_goal = img_space.sample()
```

Comment:
I think we should leave this as None. Isn't it a bug if someone tries to access `self._img_goal` before it's been set to something reasonable? We want to make that bug glaringly obvious, and not hidden just because the "type" is correct (unless you have reason to actually want to use the output of `img_space.sample())